src/ucs/sys/topo/base/topo.h

```diff
@@ -44,6 +44,16 @@ typedef struct ucs_sys_bus_id {
 typedef uint8_t ucs_sys_device_t;
 
 
+/**
+ * @ingroup UCS_RESOURCE
+ * Globally Unique System Device Identifier
+ */
+typedef struct ucs_sys_device_guid {
```

Comment:
1. the guid should be single uin64_t value
2. Maybe better call it "UID" and not "GUID" - since this is "locally unique" and not "globally unique"?